src/components/sdk/MeetingControls/AudioInputVFControl.tsx

```diff
@@ -61,6 +61,13 @@ const AudioInputVFControl: React.FC<Props> = ({
     };
   }, []);
 
+  useEffect(() => {
+    if (isVoiceFocusOn === undefined) {
+      return;
+    }
+    console.info('Updating Amazon Voice Focus display state: ', isVoiceFocusOn)
```

Comment:
Updating Amazon Voice Focus display state -> `Updating Amazon Voice Focus checkbox state`

Reply:
sure, will update it